### ISSUE TITLE
'save as' and 'open' styles

### ISSUE THREAD

**Issue by Forna73** [mapbox/mapbox-studio-classic#1509]
Ok!
I'm here again... updated to v 0.3.3 and the problem still there.
I create a style and when I save it which ever is the path I browse in it saves the style in the C: partition.
If I move the saved style to the path I need and then I try to open it, it says:

Couldn't open file at C:\Users\Stefano\Documents...\project.yml

It's becoming boring.

**Comment by wilhelmberg** [comment]
@Forna73 the fix didn't make it into the `0.3.3` release. Should come with the next one.

In the meantime could try to replace 

```
C:\Program Files\mapbox-studio\resources\app\lib\middleware.js
```

with this:
https://raw.githubusercontent.com/mapbox/mapbox-studio-classic/b89994507338ae72540ebad74e9287d911420dbc/lib/middleware.js

and tell me, if it works for you?

Don't forget to make a backup of your existing `middleware.js` before replacing.

**Comment by Forna73** [comment]
I'll let you know!

2015-11-23 16:27 GMT+01:00 Wilhelm Berg notifications@github.com:

**Comment by Forna73** [comment]
Hi!
No luck for me...
I opened a .txt file and copied all the script and them saved as .js

I'll wait  for the fix.

2015-11-23 16:27 GMT+01:00 Wilhelm Berg notifications@github.com:

**Comment by springmeyer** [comment]
note: the fix @BergWerkGIS thought would address this was https://github.com/mapbox/mapbox-studio-classic/commit/b89994507338ae72540ebad74e9287d911420dbc and will be available in upcoming v0.3.4. So the next action is to have @Forna73 try and replicate again once v0.3.4 is officially released.

**Comment by springmeyer** [comment]
@Forna73 v0.3.4 is released now - can you please test again? If you hit the same issue please create a new ticket with details.
